### PR TITLE
feat(mls-conversation): show protocol info in conversation details (AR-2157)

### PR DESCRIPTION
**Pull request by mchenani**
<table>
<td>
  <a href="https://wearezeta.atlassian.net/browse/AR-2157" title="AR-2157" target="_blank">TaskAR-2157</a>  [AR] Protocol and Cyphersuite indication in the Group properties
  </td></table>
  <br />
 

----
#### PR Submission Checklist for internal contributors

- The **PR Title**
  - [x] conforms to the style of semantic commits messages¹ supported in Wire's Github Workflow²
  - [x] contains a reference JIRA issue number like `SQPIT-764`
  - [x] answers the question: _If merged, this PR will: ..._ ³

- The **PR Description**
  - [ ] is free of optional paragraphs and you have filled the relevant parts to the best of your ability
----

# What's new in this PR?

Exposed protocol info in the Group Details screen.

### Dependencies (Optional)
Needs to be merged with Kalium.

Needs releases with:

- [x] GitHub link to other pull request
https://github.com/wireapp/kalium/pull/808

### Testing
Create an mls group, in the group details you should be able to see the protocol info.

#### Test Coverage (Optional)

- [ ] I have added automated test to this contribution

----
#### PR Post Submission Checklist for internal contributors (Optional)

 - [ ] Wire's Github Workflow has automatically linked the PR to a JIRA issue
----
#### PR Post Merge Checklist for internal contributors

 - [ ] If any soft of configuration variable was introduced by this PR, it has been added to the relevant documents and the CI jobs have been updated.
----
##### References
1. https://sparkbox.com/foundry/semantic_commit_messages
1. https://github.com/wireapp/.github#usage
1. E.g. `feat(conversation-list): Sort conversations by most emojis in the title #SQPIT-764`.

### CODE REVIEW

**Comment by AndroidBob** [comment]
Build [1](https://10.10.124.134/job/wire-android-reloaded/job/PR-870/1/) **failed**.

**Comment by AndroidBob** [comment]
Build [2](https://10.10.124.134/job/wire-android-reloaded/job/PR-870/2/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4184031)(4184031)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4184031)(4184031)-dev-release.apk)

**Comment by AndroidBob** [comment]
Build [3](https://10.10.124.134/job/wire-android-reloaded/job/PR-870/3/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4185239)(4185239)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4185239)(4185239)-dev-release.apk)

File: app/build.gradle.kts [hunk 1]
```diff
@@ -99,6 +99,7 @@ dependencies {
     implementation(Libraries.Kotlin.stdLib)
     implementation(Libraries.appCompat)
     implementation(Libraries.ktxCore)
+    implementation(Libraries.ktxDateTime)
```
Comment (MohamadJaara) [inline app/build.gradle.kts:102]: Is ktxDataTime needed anywhere in this PR ?, if not, I think we can remove this dependency

**Review by MohamadJaara — CHANGES_REQUESTED** [review]
Small question about adding ktxDataTime dependency and it it's needed, other that that looks great 🚀

File: app/build.gradle.kts (hunk 1 above)
Comment (mchenani) [reply app/build.gradle.kts:102]: we're have an Instant object in MLS Protocol info that we want to show it in the group details. so yes we need it.

**Comment by AndroidBob** [comment]
Build [4](https://10.10.124.134/job/wire-android-reloaded/job/PR-870/4/) **succeeded**.

The build produced the following APK's:
- [com.wire.android-vv0.0.1(4185737)(4185737)-dev-release.apk](https://z-lohika.s3-eu-west-1.amazonaws.com/megazord/android/reloaded/dev/release/com.wire.android-vv0.0.1(4185737)(4185737)-dev-release.apk)

**Review by MohamadJaara — APPROVED** [review]

**Pull request merged**